apps/livreGUI/stimuliController/StimuliController.cpp

```diff
@@ -222,23 +217,21 @@ struct StimuliController::Impl
 public:
 
     ::monsteer::Simulator* _simulator;
+    zeq::Subscriber* _selectionSubscriber;
```

Comment:
The whole stimuliController will be gone, I'll amend the commit with the sha1 from https://github.com/BlueBrain/Monsteer/pull/6 once it's merged